db/migrate/20230724205840_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] A post belongs to the `author` Kindly consider adding a reference to the author instead of the user so the relations are set up properly ``t.references :author, foreign_key: { to_table: 'users' }``